backend/api/Controllers/RobotController.cs

```diff
@@ -314,4 +314,64 @@ public async Task<ActionResult<IList<VideoStream>>> GetVideoStreams([FromRoute]
             throw;
         }
     }
+
+    /// <summary>
+    /// Pause current mission
+    /// </summary>
+    /// <remarks>
+    /// <para> This query pauses the currently executing mission for a robot </para>
+    /// </remarks>
+    [HttpPost]
+    [Route("{robotId}/pause/")]
+    [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<string>> PauseMission([FromRoute] string robotId)
+    {
+        try
+        {
+            var response = await _isarService.PauseMission();
```

Comment:
If ISAR returns a non-200 status code we might want to give a 503 bad gateway instead of a generic 500

Reply:
Went for 502 if something happens with ISAR. Could be granulated more later I think but maybe this is enough for now?